openassessment/xblock/staff_assessment_mixin.py

```diff
@@ -27,6 +28,26 @@ def staff_assessment_exists(self, submission_uuid):
 
         return staff_api.get_latest_staff_assessment(submission_uuid) is not None
 
+    def get_user_email_by_submission_uuid(self, submission_uuid):
```

Comment:
Seems it copy paste error.

Reply:
I created new Mixin with helper method and removed these duplicate methods.